hsmd/hsmd_wire.csv

```diff
@@ -20,7 +20,7 @@ msgdata,hsmd_init,dev_force_channel_secrets_shaseed,?sha256,
 msgtype,hsmd_init_reply,111
 msgdata,hsmd_init_reply,node_id,node_id,
 msgdata,hsmd_init_reply,bip32,ext_key,
-msgdata,hsmd_init_reply,bolt12,point32,
```

Comment:
This will break VLS compatibility, unless @devrandom has an idea on how we can work around this somehow.

Reply:
Hmm, the dangers of experimental code :(

We can put in a dummy here, and simply append a new field. Or we can leave it and derive the bolt12 key separately?

Reply:
True, VLS is doing a great job at keeping in sync, so shouldn't be much of an issue. Maybe we can add a wire-version field we bump when changing the proto, or a hash of the csv file we generate from? That way we could programmatically detect incompatibilities. Just an idea going forward.

Reply:
I think it's cleaner if we use a whole new message each time. Some code duplication, but it's very simple to see what's going on.

I'll add this...